src/sass/chrome.scss

```diff
@@ -25,7 +25,7 @@ $pf-global--image-path: './assets/images/';
 
 @import '~@redhat-cloud-services/frontend-components/Spinner/spinner.css';
 @import '~@redhat-cloud-services/frontend-components/Skeleton/skeleton.css';
-@import '~@redhat-cloud-services/frontend-components/ConditionalFilter/onditionalFilter.css';
+@import '~@redhat-cloud-services/frontend-components/ConditionalFilter/conditional-filter.css';
```

Comment:
This css should not be required anymore. Was the style broken without this?

Reply:
Nope, the link just throws 404.. should I remove it?

Reply:
Yeah, try to remove it. The v3 components inject styling with components. This is theoretically rendundant. So is the stuff above.

Reply:
Should I remove styles for all frontend components?

Reply:
Yeah, I should. Let me try it.

Reply:
Looks OK locally.